src/main/java/seedu/duke/command/ListCommandsCommand.java

```diff
@@ -1,5 +1,7 @@
 package seedu.duke.command;
 
+import seedu.duke.Ui;
+
 import java.util.Map;
 
 public class ListCommandsCommand extends Command {
```

Comment:
can consider renaming this function to avoid confusion with ListCommand

Reply:
i changed to HelpCommand